<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideShowEditor</class>
 <widget class="QWidget" name="SlideShowEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slide Show Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="softprojector.qrc">
    <normaloff>:/icons/icons/slideshow_edit.png</normaloff>:/icons/icons/slideshow_edit.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Slide Show Title:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditTitle"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Slide Show Info:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="plainTextEditInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" rowspan="4">
    <widget class="QListWidget" name="listWidgetSlides">
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="spacing">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAddImages">
       <property name="text">
        <string>Add Pictures</string>
       </property>
       <property name="icon">
        <iconset resource="softprojector.qrc">
         <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemoveImage">
       <property name="text">
        <string>Remove Picture</string>
       </property>
       <property name="icon">
        <iconset resource="softprojector.qrc">
         <normaloff>:/icons/icons/remove.png</normaloff>:/icons/icons/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMoveUp">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="softprojector.qrc">
         <normaloff>:/icons/icons/up.png</normaloff>:/icons/icons/up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMoveDown">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="softprojector.qrc">
         <normaloff>:/icons/icons/down.png</normaloff>:/icons/icons/down.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelPreview">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>300</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Picture Preview</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelPixInfo">
     <property name="text">
      <string>Picture Information</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>74</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditTitle</tabstop>
  <tabstop>plainTextEditInfo</tabstop>
  <tabstop>listWidgetSlides</tabstop>
  <tabstop>pushButtonAddImages</tabstop>
  <tabstop>pushButtonRemoveImage</tabstop>
  <tabstop>pushButtonMoveUp</tabstop>
  <tabstop>pushButtonMoveDown</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="softprojector.qrc"/>
 </resources>
 <connections/>
</ui>
